src/core/lwt_pool.ml

```diff
@@ -20,26 +20,21 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  *)
 
-(* [Lwt_sequnece] is deprecated to prevent users from using it, but it is used
+(* [Lwt_sequence] is deprecated to prevent users from using it, but it is used
```

Comment:
Argh, thanks for catching this. I copy/pasted this error into a whole bunch of files.